Lombiq.Privacy.csproj

```diff
@@ -57,4 +57,8 @@
     <PackageReference Include="Lombiq.HelpfulLibraries.OrchardCore" Version="4.2.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\Utilities\Lombiq.Gulp.Extensions\Lombiq.Gulp.Extensions.csproj" />
+  </ItemGroup>
```

Comment:
Did adding this reference actually help? Can you point me to the run where this fixed the build?

Reply:
I can't for sure, becuase the problem was inconsistent anyway. But I think it's good to have regardless whether it helped or not becuase this project actually uses Gulp. I just moved it to the `<ItemGroup Condition="'$(NuGetBuild)' != 'true'">`.

Reply:
Fair enough. I agree.